packages/frontend-core/src/fetch/ViewV2Fetch.js

```diff
@@ -55,12 +48,6 @@ export default class ViewV2Fetch extends DataFetch {
       this.options.sortOrder = definition.sort.order
     }
 
-    const parsed = utils.processSearchFilters(filter)
-
-    if (!parsed?.groups?.length && definition.query?.groups?.length) {
-      this.options.filter = definition.query
-    }
```

Comment:
We never want to set the filter to the query definition, as that is redundant and causes the filters to be sent up in the search request.